modules/bitgo/src/bitgo.ts

```diff
@@ -6,17 +6,22 @@
 import pjson = require('../package.json');
 import * as _ from 'lodash';
 
-import GlobalCoinFactory from './v2/coinFactory';
-import { BaseCoin, common } from '@bitgo/sdk-core';
+import { BaseCoin, CoinFactory, common } from '@bitgo/sdk-core';
 import { BitGoAPI, BitGoAPIOptions } from '@bitgo/sdk-api';
+import { createTokenMapUsingTrimmedConfigDetails, TrimmedAmsTokenConfig } from '@bitgo/statics';
+import { GlobalCoinFactory, registerCoinConstructors } from './v2/coinFactory';
 
-export type BitGoOptions = BitGoAPIOptions;
+export type BitGoOptions = BitGoAPIOptions & {
+  useAms?: boolean;
+};
 
 export class BitGo extends BitGoAPI {
+  private coinFactory: CoinFactory;
+  private useAms: boolean;
```

Comment:
for consistent nomenclature
```suggestion
  private _coinFactory: CoinFactory;
  private _useAms: boolean;
```